targets/mega65/uart_monitor.c

```diff
@@ -66,18 +66,19 @@ static void _umon_write_size_panic ( void )
 	DEBUGPRINT("UARTMON: warning: too long message (%d/%d), cannot fit into the output buffer!" NL, umon_write_pos, UMON_WRITE_BUFFER_SIZE);
 }
 
-static void m65mon_show_regs ( void )
+void m65mon_show_regs ( void )
 {
 	Uint8 pf = cpu65_get_pf();
 	umon_printf(
 		"\r\n"
-		"PC   A  X  Y  Z  B  SP   MAPL MAPH LAST-OP     P  P-FLAGS   RGP uS IO\r\n"
+		"PC   A  X  Y  Z  B  SP   MAPH MAPL LAST-OP     P  P-FLAGS   RGP uS IO\r\n"
 		"%04X %02X %02X %02X %02X %02X %04X "		// register banned message and things from PC to SP
 		"%04X %04X %02X       %02X %02X "		// from MAPL to P
-		"%c%c%c%c%c%c%c%c ",				// P-FLAGS
+		"%c%c%c%c%c%c%c%c \r\n"				// P-FLAGS
+		",0777%04X\r\n", // TODO: single line of disassembly
```

Comment:
Here's the line with the extra comma.

Normally, on hardware, 'r' command would show output like this:
```
<dbg>r

PC   A  X  Y  Z  B  SP   MAPH MAPL LAST-OP In     P  P-FLAGS   RGP uS IO ws h RECA8LHC
E1AF FF 00 FD 00 00 01F4 B300 E300 6C4A03  00     A4 N.E..I.. ...P 46 -  00 - ..c..l.c
,0777E1AF  B0 31     BCS   $E1E2
```

One of our tools (I think 'm65') is very sensitive in looking for that comma, to indicate that a breakpoint was hit.